### PR TITLE
Py Organiser

### LINKED ISSUE #317: An Organiser application using Python and Qt5
# Description
  - New project organizer application using python and Qt5
  - Organiser application to plan your days and todos

## Type of issue
- NOTE: *These boxes can be checked using **`[X]`***

- [X] Feature (New Script)
- [ ] Bug
- [X] Documentation

## Checklist:

- [X] I have read the project guidelines.
- [X] I have checked all the existing projects, before submitting a new project issue.
- [X] I have checked previous issues to avoid duplicates.
- [X] This issue will be meaningful for the project.

### PR DESCRIPTION
**Pull request by swaroopmaddu**
# Description

   - Add plans to your day
   - Organise your day
   - Keep track about all meetings and todo

Fixes #317

## Type of change

Choosing one or more options from the following as per the nature of your Pull request.
- NOTE: *These boxes can be checked using **`[X]`***

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation Update

# Checklist:
Please tick all the boxes that are fulfilled by your Pull Request.

- [X] I have named my files and folder, according to this project's guidelines.
- [X] My code follows the style guidelines of this project.
- [X] My Pull Request has a descriptive title. (not a vague title like `Update index.md`)
- [X] I have commented on my code, particularly in hard-to-understand areas.
- [X] I have created a helpful and easy to understand `README.md`, according to the given [`README_TEMPLATE.`](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md)
- [X] I have included a requirements.txt file (if external libraries are required.)
- [X] My changes do not produce any warnings.
- [X] I have added a working sample/screenshot of the script.
demo

### CODE REVIEW

File: Scripts/Miscellaneous/PyOrganiser/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    - Organise your day
+    - Keep track abouts all meetings and todos
+    
+# Installing
+
+  `pip install requirements.txt`
+  
+# Demo
+
+ <img src="demo.png"/>
```
Comment (chavarera) [inline Scripts/Miscellaneous/PyOrganiser/README.md:13]: Add Author Name in Readme.md files

File: Scripts/Miscellaneous/PyOrganiser/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Organiser application
+
+    - Add plans to your day
+    - Organise your day
+    - Keep track abouts all meetings and todos
+    
+# Installing
+
+  `pip install requirements.txt`
+
```
Comment (chavarera) [inline Scripts/Miscellaneous/PyOrganiser/README.md:10]: Add Steps for converting UI file to python file

**Review by chavarera — CHANGES_REQUESTED** [review]
place all  media  files at one directory

Suggested names
```
media
content
icon
img
```

File: Scripts/Miscellaneous/PyOrganiser/README.md (hunk 1 above)
Comment (swaroopmaddu) [reply Scripts/Miscellaneous/PyOrganiser/README.md:13]: Ok i'll make changes

File: Scripts/Miscellaneous/PyOrganiser/README.md (hunk 2 above)
Comment (swaroopmaddu) [reply Scripts/Miscellaneous/PyOrganiser/README.md:10]: No need to convert Ui file to Py

**Comment by swaroopmaddu** [comment]
@chavarera can you check my repo once cause I have to override a method so have to pass an argument without using it.

**Comment by swaroopmaddu** [comment]
Failing DeepSource: Python because of unused arguments @chavarera

**Pull request merged**